### ISSUE TITLE
[Easy] Need to change `@blitzjs/core` to `blitz` in new app template

### ISSUE THREAD

**Issue by flybayer** [blitz-js/blitz#378]
### What is the problem?

The below change needs to be made in

`packages/cli/templates/app/app/pages/_document.tsx`

```diff
--import {Document, Html, DocumentHead, Main, NextScript /*DocumentContext*/} from '@blitzjs/core'
++import {Document, Html, DocumentHead, Main, NextScript /*DocumentContext*/} from 'blitz'
```

**Note:** this is an easy change, so let someone take this who hasn't contributed to Blitz before

**Comment by developerfred** [comment]
I wiil do

**Comment by francocorreasosa** [comment]
@flybayer this one should be closed right? (PR #379)

**Comment by flybayer** [comment]
@francocorreasosa yes!
